eden/fs/config/EdenConfig.h

```diff
@@ -17,7 +17,7 @@
 #include <folly/portability/SysTypes.h>
 #include <folly/portability/Unistd.h>
 
-#include "common/rust/shed/hostcaps/hostcaps.h"
+//#include "common/rust/shed/hostcaps/hostcaps.h"
```

Comment:
Instead of removing it, maybe

```suggestion
#ifdef EDEN_HAVE_SERVICEROUTER
#include "common/rust/shed/hostcaps/hostcaps.h"
#endif
```